File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.py
def solution(myString, pat):
    myString = myString.replace('A', 'X')
    myString = myString.replace('B', 'A')
    myString = myString.replace('X', 'B')
    
    if pat in myString:
        return 1
    else:
        return 0

File: 프로그래머스/0/181864. 문자열 바꿔서 찾기/문자열 바꿔서 찾기.js
function solution(myString, pat) {
    myString = myString.replaceAll('A', 'X');
    myString = myString.replaceAll('B', 'A');
    myString = myString.replaceAll('X', 'B');
    
    if (myString.includes(pat)) {
        return 1;
    }
    
    return 0;
}